subscription_web_access/models/res_partner.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2021 Coop IT Easy SC
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    is_web_subscribed = fields.Boolean(
+        compute="_compute_subscription_status",
+        store=True,
+    )
+
+    subscriber = fields.Boolean(
+        compute="_compute_subscription_status",
+        store=True,
+    )
```

Comment:
the naming of these fields is strange. i would name the first `has_(ongoing_)subscription` and the second `has_(ongoing_)paying_subscription` (or something similar). what do you think? the `web` part in the name doesn’t really make sense on its own.

Reply:
Since this module is only made for Médor, and the names match Médor's use case, let's keep this as is.